rpa-builder/6.5/modules/ROOT/pages/toolbox-aws-operations-signature-detection-with-aws.adoc

```diff
@@ -0,0 +1,86 @@
+= Signature Detection with AWS
+
+Detect signatures in a PNG, JPEG, or PDF file using the AWS Signature Detection services. You can configure the action step to expect one or more signatures and adjust the confidence level for the detection. 
+
+include::partial$aws-doc-processing.adoc[tag=specific]
+
+== Inbound Variables
+
+* *Directory Path* 
+* *File Name*
+* *Password* 
+* *Read text from page* 
+* *Read text to page* 
+* *AWS Credentials* 
+* *Client Id* 
+* *Client Secret* 
+* *Session Token*
+* *Use Session Token* 
+* *Region Endpoint* 
+* *Json Path*
+
+== Outbound Variables 
+
+* *Raw Json* 
++
+The complete AWS response.
+* *Normalized Json* 
++
+The parsed JSON response including page number, signature name, and confidence score.
+* *Signature Found* 
++
+A boolean value to confirm if a signature was found. 
+* *Signature Count* 
++
+The number of signatures found. 
+
+== Properties 
+
+* *Authentication Settings*
+** *Region Endpoint* 
++
+The region endpoint to use for the AWS Service.
+** *Use Session Token* 
++
+Enables using a session token for authentication. 
```

Comment:
**Question:** These are just two settings of the common AWS authentication settings present in every AWS action step. Shouldn't these settings be a partial?